scout/commands/query/query_command.py

```diff
@@ -1,10 +1,19 @@
 import logging
-
+from flask.cli import with_appcontext
+from scout.server.extensions import store
 import click
 
 log = logging.getLogger(__name__)
 
-@click.command('hgnc', short_help='Check if a gene exist')
+@click.group()
```

Comment:
Yes, me like! No dependencies on this I guess? Or was the grouping implicit before?